tests/utils/utils.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package tests
+
+import (
+	"bufio"
+	"context"
+	"os"
+	"os/exec"
+
+	"github.com/ethereum/go-ethereum/log"
+	. "github.com/onsi/gomega"
+)
+
+func RunRelayerExecutable(ctx context.Context, relayerConfigPath string) (*exec.Cmd, context.CancelFunc) {
+	cmdOutput := make(chan string)
+
+	// Run awm relayer binary with config path
+	var relayerContext context.Context
+	relayerContext, relayerCancel := context.WithCancel(ctx)
+	relayerCmd := exec.CommandContext(relayerContext, "./build/awm-relayer", "--config-file", relayerConfigPath)
+
+	// Set up a pipe to capture the command's output
+	cmdReader, _ := relayerCmd.StdoutPipe()
```

Comment:
Are we intentionally ignoring the error here for a reason? If so, please leave a comment.

Reply:
Nope, this was an oversight. Added that check in.